docs/USAGE.md

```diff
@@ -50,7 +50,7 @@ Usage of go-earlybird:
   -file string
     	Output file -- e.g., 'go-earlybird --file=/home/jdoe/myfile.csv'
   -format string
-    	Output format [ console | json | csv ] (default "console")
+    	Output format [ console | json | csv ] (default "console"). To print finding in console with json format, use `-withConsole`
```

Comment:
is format console not suppose to work??? I do not think the msg in the end is needed if the withConsole already mentions what it does??